backend/src/main/java/gov/cdc/usds/simplereport/api/testresult/TestResultMutationResolver.java

```diff
@@ -27,15 +27,17 @@ public class TestResultMutationResolver {
   @MutationMapping
   @AuthorizationConfiguration.RequireGlobalAdminUser
   public boolean resendToReportStream(
-      @Argument List<UUID> testEventIds, @Argument boolean fhirOnly) {
+      @Argument List<UUID> testEventIds, @Argument boolean fhirOnly, @Argument boolean covidOnly) {
     testEventRepository
         .findAllByInternalIdIn(testEventIds)
         .forEach(
             testEvent -> {
               if (!fhirOnly) {
                 testEventReportingService.report(testEvent);
               }
-              fhirReportingService.report(testEvent);
+              if (!covidOnly) {
```

Comment:
worried about the ergonomics of this method without a default "send" case. as is, we need to pass in one of `covidOnly` or `fhirOnly` for this to work, which functionally means that the caller needs to know about the method internals (the specifics of fhir vs covid only sends), which feels like a leaky abstraction to me. Think we should add an else to do a report on both pipelines (or some other default case if I'm misunderstanding) if neither is passed in. 

Also, think if we swap the conditionals this will read better: 
```
if (fhirOnly) {fhirReportingService.report(testEvent);}
if (covidOnly) {testEventReportingService.report(testEvent);}
```

Reply:
Is there a case where a caller would ever not know about the function contract? Aren't we the only ones who'd be hitting it and be the ones who know the semantic differences of how the tests are sent?

Reply:
yes on both points, but the point here is that the caller _shouldn't_ need to know about the difference since in my mind, the mechanics of how we send info to ReportStream should be abstracted, at least by default. "resend" here is a "what are we doing" vs "resend via the covid pipeline" is a "what are we doing / how we're doing it", and requiring the later by default is a sign to me that we should tweak how this function is behaving by default since at that point, [the function is existing at two levels of abstraction](https://softwareengineering.stackexchange.com/questions/110933/how-to-determine-the-levels-of-abstraction). think it's fine for us to have overrides in the event that we want it to do both things, but by default, this function should "just resend to ReportStream" without any further configuration.

Reply:
Good point for sending to both pipelines by default if neither are provided!